site/pages/dept-fr.hbs

```diff
@@ -6,73 +6,39 @@
-			</div>
-			<div class="col-lg-9 input-group">
-				<input type="text" class="form-control" size="10" id="deptSearch" placeholder="Ministère ou organisme" />
-				<span class="input-group-btn">
-		            <button type="submit" id="search-submit" class="btn btn-primary" name="search-submit"><span class="glyphicon-search glyphicon"></span><span class="wb-inv">Rechercher ministères et organismes</span></button>
-		        </span>
-			</div>
-		</div>
-		</form>
-	</div>
+<div class="row profile">
+  <div class="col-md-12">
+    <p class="pagetag">Taper le nom ou l’abréviation d’un ministère pour trouver les ministères, les organismes gouvernementaux et de service spécial et les sociétés d'État actuels du gouvernement du Canada.</p>
+  </div>
 </div>
-<hr />
-<ul class="list-unstyled lst-spcd">
-	<li><a class="h3" href="#">Now du Ministère</a><br />
-		<span class="badge badge-dept">8</span> Fonctionne aussi avec…
-		<ul class="mrgn-bttm-lg mrgn-lft-lg">
-			<li><a href="#">Nom de l’institution</a></li>
-			<li><a href="#">Nom de l’institution</a></li>
-			<li><a href="#">Nom de l’institution</a></li>
-			<li><a href="#">Nom de l’institution</a></li>
-			<li><a href="#">Nom de l’institution</a></li>
-			<li><a href="#">Nom de l’institution</a></li>
-			<li><a href="#">Nom de l’institution</a></li>
-			<li><a href="#">Nom de l’institution</a></li>
-		</ul>
-	</li>
-	<li><a class="h3" href="#">Now du Ministère</a><br />
-		<span class="badge badge-dept">6</span> Fonctionne aussi avec…
-		<ul class="mrgn-bttm-lg mrgn-lft-lg">
-			<li><a href="#">Nom de l’institution</a></li>
-			<li><a href="#">Nom de l’institution</a></li>
-			<li><a href="#">Nom de l’institution</a></li>
-			<li><a href="#">Nom de l’institution</a></li>
-			<li><a href="#">Nom de l’institution</a></li>
-			<li><a href="#">Nom de l’institution</a></li>
-		</ul>
-	</li>
-	<li><a class="h3" href="#">Now du Ministère</a><br />
-		<span class="badge badge-dept">16</span> Fonctionne aussi avec…
-		<ul class="mrgn-bttm-lg mrgn-lft-lg">
-			<li><a href="#">Nom de l’institution</a></li>
-			<li><a href="#">Nom de l’institution</a></li>
-			<li><a href="#">Nom de l’institution</a></li>
-			<li><a href="#">Nom de l’institution</a></li>
-			<li><a href="#">Nom de l’institution</a></li>
-			<li><a href="#">Nom de l’institution</a></li>
-			<li><a href="#">Nom de l’institution</a></li>
-			<li><a href="#">Nom de l’institution</a></li>
-			<li><a href="#">Nom de l’institution</a></li>
-			<li><a href="#">Nom de l’institution</a></li>
-			<li><a href="#">Nom de l’institution</a></li>
-			<li><a href="#">Nom de l’institution</a></li>
-			<li><a href="#">Nom de l’institution</a></li>
-			<li><a href="#">Nom de l’institution</a></li>
-			<li><a href="#">Nom de l’institution</a></li>
-			<li><a href="#">Nom de l’institution</a></li>
-		</ul>
-	</li>
-</ul>
\ No newline at end of file
+<table class="wb-tables table" data-wb-tables="{ &quot;paging&quot;: false }">
```

Comment:
Now we have a "filter" plugin, I would recommend to use it for that page instead of relying on the data table search feature. Make that suggestion to the policy, they might now aware of that plugin http://wet-boew.github.io/wet-boew/demos/filter/filter-en.html